npm_and_yarn/Dockerfile

```diff
@@ -7,7 +7,7 @@ ARG PNPM_VERSION=8.9.2
 ARG YARN_VERSION=3.6.4
 
 # See https://github.com/nodesource/distributions#installation-instructions
-ARG NODEJS_VERSION=18.x
+ARG NODEJS_VERSION=20.x
```

Comment:
unrelated, but what do you think about moving the `.x` from `NODEJS_VERSION` to line 20 so it's `node_${NODEJS_VERSION}.x`

Reply:
That sounds reasonable, thanks!